src/CMakeLists.txt

```diff
@@ -146,28 +152,24 @@ target_compile_definitions( ${TARGETLIB} PUBLIC VERSION="${VERSION}" )
 # Linking
 #
 
-# Find yui during a combined build
-target_link_directories( ${TARGETLIB} BEFORE PUBLIC ../../libyui/build/src )
+# https://cmake.org/cmake/help/latest/command/link_directories.html suggests to use target_link_libraries
```

Comment:
This whole block (line 155..161) is not needed for libyui-gtk. We did that for libyui-qt etc. so we can do completely local builds of libyui, libyui-qt, libyui-qt-pkg, libyui-qt-graph, libyui-ncurses, libyui-ncurses-pkg, libyui-rest-api, libyui-rest-api-qt, libyui-rest-api-ncurses without the need of `make install` all the time: This would simply prefer freshly built libraries in ../../libyui/build/...  

But this is not needed or desired here because this subproject (libyui-gtk) is always built separately. It doesn't make any sense to add more link directories (i.e. `-L../../libyui/build/...` parameters) here.

Reply:
First of all, thanks for starting reviewing all. I will take a look of your comments one by one, regarding local deps block i'm evaluating if leaving it because it could be usefull for packagers since it could be easier to add libyui-gtk and libyui-mga plugins as a part of the libyui SRPM (or deb) to build all in one shot and simplify the libyui-bindings building too.

Reply:
Agreed. I added an exception to libyui-mga that could be built into libyui tree so that libyui-bindings can find its includes and lib as well as libyui ones. Will talk about it into another PR (for bindings) i'm opening to start discussion there too

Reply:
OK.